src/components/icon-button/IconButton.constants.ts

```diff
@@ -0,0 +1,64 @@
+import { TIconSize } from '../icon';
+import { TTextSize } from '../text';
+
+import { TIconButtonSize } from './IconButton.types';
+
+const DEFAULT_ICON_BUTTON_SIZE: TIconButtonSize = 'standard';
+
+const ICON_BUTTON_SIZES: TIconButtonSize[] = [
+    'xsmall',
+    'small-compact',
+    'small',
+    'standard',
+    'large',
+];
+
+const ICON_BUTTON_TEXT_SIZES: Record<TIconButtonSize, TTextSize> = {
```

Comment:
wonder if it will be better to group by IconSize and define the props for each one. Maybe it will be more visible? And I know it will help with binding in the story.

Reply:
See if what I did is what you were thinking. :)